marklogic-data-hub-central/ui/src/components/search-result/search-result.tsx

```diff
@@ -50,17 +50,12 @@ const SearchResult: React.FC<Props> = (props) => {
     }
 
     if ((props.item.format === 'json' || props.item.format === 'xml') && props.item.hasOwnProperty('sources')) {
-        if (!props.item.sources.length) {
-            // content data does not exist in payload
-            setAlertMessage('Error', 'No instance information in payload');
+        if (Array.isArray(props.item.sources)) {
```

Comment:
Do we know of examples where sources is an object and not an array? Pinging @srinathgit  and @ryanjdew  - that would violate the schema, and hopefully that's not something we've created. I think it's fine to have this code as a sanity check in case sources is an object somehow, but I want to make sure that we're not making sources an object somehow.

Reply:
Yes. In createHeaders, even if it is an object, it is converted to an array.
In entity-search-lib.sjs as well, sources array is returned. The check can be removed.